src/crashlytics/listTopIssues.ts

```diff
@@ -0,0 +1,63 @@
+import { Client } from "../apiv2";
+import { logger } from "../logger";
+import { FirebaseError } from "../error";
+import { crashlyticsApiOrigin } from "../api";
+import { FirebaseFilters, FirebaseFilterInterval, listTopIssuesParams, PageDetails } from "./paramInterface";
+
+const TIMEOUT = 10000;
+
+const apiClient = new Client({
+    urlPrefix: crashlyticsApiOrigin(),
+    apiVersion: "v1",
+  });
+
+export async function listTopIssues(
+    projectId: string,
+    platform: string,
+    packageName: string, 
+    issueCount: string
+): Promise<string> {
+    try {
+        const filters: FirebaseFilters = {
+            categories: [],
+            customKeys: [],
+            eventType: ["FATAL"],
+            hasCrashlyticsInsights: false,
+            manufacturerModels: [],
+            osVersions: [],
+            rollouts: [],
+        }
+
+        const now = new Date();
+        const pastDate: Date = new Date(now.getTime() - (7 * 24 * 60 * 60 * 1000));
```

Comment:
Don't need explicit type for simple assignment. Also, why is this not something that is configurable in the API call?

```suggestion
        const pastDate = new Date(Date.now() - (7 * 24 * 60 * 60 * 1000));
```